.ci/build_pipeline_definitions

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash -eu
+
+SOURCE_PATH=${SOURCE_PATH:-}
+
+if [[ -z ${SOURCE_PATH} ]]; then
+    SOURCE_PATH="$(readlink -f "$(dirname "${0}")"/..)"
+fi
+
+cat > "${SOURCE_PATH}"/.ci/pipeline_definitions << EOF
+ops-toolbelt:
+  template: 'default'
+  base_definition:
+    repo: ~
+    traits:
+      version:
+        preprocess:
+          'inject-commit-hash'
+        inject_effective_version: true
+      publish:
+        dockerimages:
+          ops-toolbelt-gardenctl:
+            registry: 'gcr-readwrite'
+            image: 'eu.gcr.io/gardener-project/gardener/ops-toolbelt-gardenctl'
+            dockerfile: ops-toolbelt-gardenctl.dockerfile
+            inputs:
+              steps:
+                build: ~
+          ops-toolbelt:
+            registry: 'gcr-readwrite'
+            image: 'eu.gcr.io/gardener-project/gardener/ops-toolbelt'
+            dockerfile: ops-toolbelt.dockerfile
+            inputs:
+              steps:
+                build: ~
+EOF
+
+for i in `cat "${SOURCE_PATH}"/.ci/k8s_versions`
+do
+    cat >> "${SOURCE_PATH}"/.ci/pipeline_definitions << EOF
+          ops-toolbelt-aws-k$i:
+            registry: 'gcr-readwrite'
+            image: 'eu.gcr.io/gardener-project/gardener/ops-toolbelt-aws-k$i'
```

Comment:
I don't think we should have the patch version in the image name. Major and minor version should be sufficient